R/ggMarginalHelpers.R

```diff
@@ -245,4 +245,10 @@ addRightMargPlot <- function(ggMargGrob, right, size) {
                                 b = panelPos[["b"]], r = ncol(gt), l = ncol(gt),
                                 z = Inf, clip = "on", name = "rightMargPlot")
   gt
+}
+
+getTitleGrob <- function(p) {
+  grobs <- ggplot2::ggplotGrob(p)$grobs
+  gind <- sapply(grobs, function(x) grepl(pattern = "plot.title", x$name))
```

Comment:
You're right. And yes, since this isn't using an explicit ggplot API, it's a good idea to unit test for it